### PR TITLE
Added some new tests for a few webBluetooth method.

### PR DESCRIPTION
**Pull request by fokinv**
We made some new tests for WebBluetooth.

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14614)

### CODE REVIEW

**Comment by KiChjang** [comment]
Nit: The commit message has a typo on Bluetooth.

File: tests/wpt/mozilla/tests/mozilla/bluetooth/requestDevice/acceptAllDevices/device-with-no-name.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/_mozilla/mozilla/bluetooth/bluetooth-helpers.js"></script>
+<script>
+'use strict';
+promise_test(() => {
+    window.testRunner.setBluetoothMockDataSet(adapter_type.no_name_heart_rate);
+    return window.navigator.bluetooth.requestDevice({acceptAllDevices: true})
+    .then(device => assert_true(device.name === null));
```
Comment (jdm) [inline tests/wpt/mozilla/tests/mozilla/bluetooth/requestDevice/acceptAllDevices/device-with-no-name.html]: assert_equals(device.name, null)

File: tests/wpt/mozilla/tests/mozilla/bluetooth/requestDevice/acceptAllDevices/optional-services-missing.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/_mozilla/mozilla/bluetooth/bluetooth-helpers.js"></script>
+<script>
+'use strict';
+promise_test(t => {
+    window.testRunner.setBluetoothMockDataSet(adapter_type.two_heart_rate);
+    return window.navigator.bluetooth.requestDevice({acceptAllDevices: true})
+    .then(device => device.gatt.connect())
+    .then(gattServer => promise_rejects(t, 'NotFoundError' ,gattServer.getPrimaryServices()));
```
Comment (jdm) [inline tests/wpt/mozilla/tests/mozilla/bluetooth/requestDevice/acceptAllDevices/optional-services-missing.html]: nit: move the , next to the string and add a space after it.

**Review by jdm — CHANGES_REQUESTED** [review]
Looks like a good set of tests!

**Comment by jdm** [comment]
@bors-servo: delegate+
This can merge with r=jdm after squashing.

**Comment by dati91** [comment]
@jdm Valentin is already on Holiday, so if you could r+ it that would be great!

**Pull request merged**